src/cleanup.ts

```diff
@@ -25,7 +25,9 @@ export async function run(cutoff = '24h'): Promise<void> {
     await docker.pruneNetworks({filters: untilFilter})
     core.info(`Pruning containers older than ${cutoff}`)
     await docker.pruneContainers({filters: untilFilter})
-    await cleanupOldImageVersions(docker, UPDATER_IMAGE_NAME)
+    for (const image of updaterImages()) {
+      await cleanupOldImageVersions(docker, image)
+    }
```

Comment:
I suppose when customers upgrade this will leave a single monolithic updater image dangling. Is it worth keeping a reference to the old updater around for this?

Reply:
We should check how long this takes to cleanup all these images. If it's noticeably impacting the job runtime we might want to scale this down to just cleanup the images for the ecosystem we ran for. If it's not noticeable then we might as well add another step that would trigger cleaning out the monolith images. At least until we no longer support GHES versions that would use the monolith image?